### ISSUE TITLE
«ju_prepend(): Argument #2 ($head) must be of type string, null given»

### ISSUE THREAD

**Issue by javiertoledos-xumak** [justuno-com/core#381]
In file `\Justuno\Core\Framework\Plugin\Data\Form\Element\AbstractElement::afterGetElementHtml`, on line 44 there's a call to ju_prepend so if an AbstractElement doesn't have a "before_element_html" class provided, it crashes as ju_prepend doesn't accept nullable strings.

the solution would be to allow nullable strings or use null coalesce operator:

```php
    public function afterGetElementHtml(Sb $sb, string $r):string
    {
        return ju_starts_with($r, '<label class="addbefore"') ? $r : ju_prepend($r, $sb->getBeforeElementHtml() ?? '');
    }
```

**Comment by javiertoledos-xumak** [comment]
Here is the patch I used (relative to the module folder)
```
diff --git a/Framework/Plugin/Data/Form/Element/AbstractElement.php b/Framework/Plugin/Data/Form/Element/AbstractElement.php
index 672be222..512a6a86 100644
--- a/Framework/Plugin/Data/Form/Element/AbstractElement.php
+++ b/Framework/Plugin/Data/Form/Element/AbstractElement.php
@@ -41,7 +41,7 @@ class AbstractElement extends Sb {
 	 * @param Sb $sb
 	 */
 	function afterGetElementHtml(Sb $sb, string $r):string {return
-		ju_starts_with($r, '<label class="addbefore"') ? $r : ju_prepend($r, $sb->getBeforeElementHtml())
+		ju_starts_with($r, '<label class="addbefore"') ? $r : ju_prepend($r, $sb->getBeforeElementHtml() ?? '')
 	;}

```

**Comment by dmitry-fedyuk** [comment]
Fixed in [1.3.3](https://github.com/justuno-com/core/releases/tag/1.3.3).
